shared/data/surrogates.txt

```diff
@@ -0,0 +1,360 @@
+        build: noopReturnNull
+    }
+    const contentTarget = {
+        addEventListener: noopReturnThis
+    }
+    const slotTarget = {
+        get: noopReturnNull,
+        getAdUnitPath: noopReturnEmptyArray,
+        getAttributeKeys: noopReturnEmptyArray,
+        getCategoryExclusions: noopReturnEmptyArray,
+        getDomId: noopReturnEmptyString,
+        getSlotElementId: noopReturnEmptyString,
+        getTargeting: noopReturnEmptyArray,
+        getTargetingKeys: noopReturnEmptyArray
+    }
+    const gptObj = {
+        _loadStarted_: true,
+        apiReady: true,
+        pubadsReady: true,
+        cmd: [],
+        pubads: function() { return new Proxy(pubadsTarget, noopHandler) },
+        companionAds: function() { return new Proxy(companionadsTarget, noopHandler)},
+        sizeMapping: function() { return new Proxy(sizeMappingTarget, noopReturnThisHandler)},
+        content: function() { return new Proxy(contentTarget, noopHandler)},
+        defineSlot: function() { return new Proxy(slotTarget, noopReturnThisHandler)},
+        defineOutOfPageSlot: function() { return new Proxy(slotTarget, noopReturnThisHandler)},
+        defineUnit: noopReturnNull,
+        destroySlots: noop,
+        disablePublisherConsole: noop,
+        display: noop,
+        enableServices: noop,
+        getVersion: noopReturnEmptyString
+    }
+    const commandQueue = (window.googletag && window.googletag.cmd.length) ? window.googletag.cmd : []
+    gptObj.cmd.push = function(arg) {
+        if (typeof arg === 'function') {
+            try {
+                arg()
+            } catch (error) {
+            }
+        }
+        return 1
+    }
+    window.googletag = gptObj
+    while(commandQueue.length > 0) {
+        gptObj.cmd.push(commandQueue.shift())
+    }
+})()
+
+googletagmanager.com/gtm.js application/javascript
+(() => {
+    'use strict'
+    const noop = () => {}
+    const datalayer = window.dataLayer
+    window.ga = (window.ga === void 0) ? noop : window.ga
+    if (datalayer) {
+        // execute callback if exists, see https://www.simoahava.com/gtm-tips/use-eventtimeout-eventcallback/
+        if (typeof datalayer.push === 'function') {
+            datalayer.push = (obj) => {
+                if (typeof obj === 'object' && typeof obj.eventCallback === 'function') {
+                    const timeout = obj.eventTimeout || 10
+                    try {
+                        setTimeout(obj.eventCallback, timeout)
+                    } catch(error) {}
+                }
+            }
+        }
+        // prevent page delay, see https://developers.google.com/optimize
+        if (datalayer.hide && datalayer.hide.end) {
+            try {
+                datalayer.hide.end()
+            } catch (error) {}
+        }
+    }
+})()
+
+googlesyndication.com/adsbygoogle.js application/javascript
+(() => {
+    'use strict'
+})()
+
+doubleclick.net/instream/ad_status.js application/javascript
```

Comment:
ditto - extracted id is `instream` while actual id should be `ad_status.js`

<img width="551" alt="failed-to-fetch" src="https://user-images.githubusercontent.com/985504/88046167-f0bd1580-cb4f-11ea-8eb6-ccdfbed3764c.png">